### PR TITLE
Aeproject compile

### PR DESCRIPTION
**Pull request by vsavovlime**

### CODE REVIEW

File: bin/aeproject/cli-commands/compile/compile.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+async function compileAndPrint(file){
+    print('\r')
+    
+    try{
+        let client = await utils.getClient();
+        let code = readFile(file)
+        let contract = await client.contractCompile(code.toString());
+
+        print(`Contract '${file} has been succeffully compiled'`)
```
Comment (jsnewby) [inline bin/aeproject/cli-commands/compile/compile.js]: successfully

File: bin/aeproject/cli-commands/compile/compile.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+require = require('esm')(module /*, options */) // use to handle es6 import/export
+import { printError, print } from '../../../utils/print'
+const utils = require('../../utils.js');
+const readFile = require('../../../utils/helpers').readFile;
```
Comment (maptuhec) [inline bin/aeproject/cli-commands/compile/compile.js]: I think this is fine for now, but we should keep one in mind for this and "print" if we split projects.

File: bin/aeproject/utils.js [hunk 3]
```diff
@@ -31,7 +46,40 @@ const copyFileOrDir = (sourceFileOrDir, destinationFileOrDir, copyOptions = {})
   fs.copySync(sourceFileOrDir, destinationFileOrDir, copyOptions)
 }
 
+const getFiles = async function (directory, regex) {
```
Comment (maptuhec) [inline bin/aeproject/utils.js]: Fix **test.js** file to use the function from the **utils** file instead the implementation in it

Comment (vsavovlime) [reply bin/aeproject/utils.js]: Done

**Pull request merged**